hack/generated/controllers/generic_controller.go

```diff
@@ -107,20 +107,48 @@ func (options *Options) setDefaults() {
 	}
 }
 
+func RegisterWebhooks(mgr ctrl.Manager, objs []runtime.Object) []error {
```

Comment:
Is the signature for this fixed, or could you return just `error` and use `kerrors.NewAggregate()` as we have elsewhere to combine the `[]error` into one?

Reply:
Yeah we can move to use `error` in both. Have done so.